apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -429,19 +429,97 @@ type RolloutStrategy struct {
+	//
+	//   Note that with this strategy, any change made to the managed fields by agents other than
+	//   Fleet in the member cluster will be overwritten periodically. If you would like Fleet
+	//   to detect such changes and report them rather than overwritting values as soon as Fleet
+	//   finds the inconsistencies, consider using the ReportDiff strategies.
+	//
+	// * ReportDiff: Fleet uses server-side apply to apply manifests, and will report
+	//   that an object has drifted from its desired state if an inconsistency has been found
+	//   in accordance with the DiffMode setting. This is helpful in drift detection scenarios,
+	//   where you would like to identify ad-hoc changes made to placed objects in
+	//   the member clusters.
+	//
+	//   See also the comments on the DiffMode field for more information.
+	//
+	//   If a diff has been found in a field that is managed by Fleet (i.e., the field is specified
+	//   in the hub cluster manifest), consider one of the following actions:
+	//
+	//   * revert the field change in all drifted member clusters.
+	//   * update the hub cluster manifest; this will trigger Fleet to apply the latest revision
+	//     of the manifests, which will overwrite the drifted field.
+	//   * switch to the ServerSideApply strategy, which will allow Fleet to overwrite the drifted
+	//     field with the current revision of manifests.
+	//
+	//   If a diff has been found in a field that is not managed by Fleet (i.e., the field is not
+	//   specified in the hub cluster manifest), consider one of the following actions:
+	//
+	//   * remove the field from all drifted member clusters.
+	//   * update the hub cluster manifest so that the field is included in the hub cluster
+	//     manifest.
+	//
+	// For a comparison between the different strategies and some examples, please refer to the
+	// Fleet documentation.
+	//
 	// +kubebuilder:default=ClientSideApply
 	// +kubebuilder:validation:Enum=ClientSideApply;ServerSideApply;ReportDiff
 	// +kubebuilder:validation:Optional
 	Type ApplyStrategyType `json:"type,omitempty"`
 
+	// DiffMode controls the way Fleet detects inconsistencies between a resource's desired state
+	// as kept in the Fleet hub cluster and its current state.
+	//
+	// This field is in effect when the ClusterResourcePlacement is configured to use the
+	// ReportDiff apply strategy and/or the ApplyIfNoDiff takeover action.
+	//
+	// Available options include:
+	//
+	// * ApplyDiff: with this mode, Fleet considers that a placed object has drifted from its
+	//   desired state if (and only if) there are value differences in fields that are managed
+	//   by Fleet (i.e., the fields that are specified explicitly in the hub cluster manifests).
+	//   Differences in unmanaged fields will be ignored. This is the default mode.
+	//
+	//   Note that as an exception, Fleet will always check for differences in the object
```

Comment:
to keep it simple, let's not make exception

Reply:
Done 👌